quickwit/quickwit-config/resources/tests/index_config/hdfs-logs.json

```diff
@@ -54,7 +54,8 @@
         "merge_policy": {
             "type": "stable_log",
             "merge_factor": 9,
-            "max_merge_factor": 11
+            "max_merge_factor": 11,
+            "maturity_period": "48 hours"
```

Comment:
Technically, that's an immaturity period. `maturation_period` might be what we are looking for.